packages/gatsby-core-utils/src/index.ts

```diff
@@ -20,5 +20,6 @@ export { listPlugins } from "./list-plugins"
 export { createFilePath } from "./filename-utils"
 export { readConfigFile, getConfigPath } from "./utils"
 export { lock } from "./lock"
+export { murmurhash } from "./murmurhash"
```

Comment:
should we export? Maybe we shouldn't have a global export casue nodejs doesn't tree-shake

Reply:
I guess this would be something for the package as a whole, not only for the murmurhash